### DISCUSSION TITLE
Can't send mail via SMTP with fresh install (v2.1.0)

### DISCUSSION THREAD

**Discussion by stevenart** [plausible/analytics#4168] [Self-Hosted Support]
I've installed a fresh v2.1.0 on a server. Everything is working find but emails are not sending.

Here is my config:
```
MAILER_EMAIL=plausible@micromegas.me
MAILER_NAME=Plausible
SMTP_HOST_ADDR=mail.mysmtpserver.com
SMTP_HOST_PORT=25
SMTP_USER_NAME="username"
SMTP_USER_PWD="password"

```

(I also tried with: `MAILER_ADAPTER=Bamboo.Mua`, same)

When I check logs from `hosting-plausible-1` container, I see this error:
```
08:33:30.923 [error] Task #PID<0.4382.0> started from #PID<0.4381.0> terminating
** (ArgumentError) raise/1 and reraise/2 expect a module name, string or exception as the first argument, got: {:network_failure, ~c"mail", {:error, :nxdomain}}
    (bamboo 2.3.0) lib/bamboo/strategies/task_supervisor_strategy.ex:25: anonymous fn/3 in Bamboo.TaskSupervisorStrategy.deliver_later/3
    (elixir 1.16.0) lib/task/supervised.ex:101: Task.Supervised.invoke_mfa/2
Function: #Function<0.95237244/0 in Bamboo.TaskSupervisorStrategy.deliver_later/3>
    Args: []
```

Does anyone knows what's wrong?

Thanks!

**Comment by ruslandoga** [comment]
👋 @stevenart 

Please share your smtp host address. Note that right now you are using `mail` which used to point to another container in the default docker compose config for v2.0.0 but has been removed in the one for v2.1.0.

**Reply by ruslandoga** [reply]
Does your host have IPv4 enabled? The reason for nxdomain could be:
- no network connection 
- domain not resolvable (seems to be your case since there is no `mail` service in docker compose anymore)
- no IPv4 support

**Reply by stevenart** [reply]
Yes, I misunderstood what `docker restart` did. And what it did not was reload configuration file. 😓

**Comment by stevenart** [comment]
Ok, I thought "docker restart [container]" reloaded the configuration but apparently not. 
I stopped the containers and did a "docker compose up" and now it works.

(Obviously not an Docker expert, sorry. 😓)

Thanks @ruslandoga for the quick response to my ignorance ! 🙏
